build_system/seqan3-config.cmake

```
@@ -616,7 +599,7 @@ if (SEQAN3_FIND_DEBUG)
   message ("  CMAKE_INCLUDE_PATH          ${CMAKE_INCLUDE_PATH}")
   message ("  SEQAN3_INCLUDE_DIR          ${SEQAN3_INCLUDE_DIR}")
   message ("")
-  message ("  ${FIND_NAME}_FOUND                ${${FIND_NAME}_FOUND}")
+  message ("  ${CMAKE_FIND_PACKAGE_NAME}_FOUND                ${${CMAKE_FIND_PACKAGE_NAME}_FOUND}")
```

you broke the alignment 😱

---

No I didn't, because it will expand to a six letter:

```suggestion
  message ("  sEqAn3_FOUND                ${${CMAKE_FIND_PACKAGE_NAME}_FOUND}")
  message ("  SEQAN3_HAS_ZLIB             ${ZLIB_FOUND}")
```

---

⚡️ Ahh, now I see the intent. 👍